batchop/env.py

```diff
@@ -45,6 +46,7 @@ def run_migrations_offline() -> None:
         literal_binds=True,
         dialect_opts={"paramstyle": "named"},
         render_as_batch=True,
+        transactional_ddl=True,
```

Comment:
Some questions are left.

* Even if I made `transactional_ddl` False, a temporary table not remains. So what this option does?
* According to the [documentation](https://alembic.sqlalchemy.org/en/latest/api/ddl.html?highlight=transactional_ddl#module-alembic.ddl.mysql), the default `transactional_ddl` value for MySQL is False. Is there something side-effect to MySQL?